bcs-runtime/bcs-k8s/kubernetes/apis/networkextension/v1/ingress_types.go

```diff
@@ -40,6 +40,10 @@ const (
 	WorkloadKindStatefulset = "statefulset"
 	// WorkloadKindGameStatefulset kind name of workload game statefulset
 	WorkloadKindGameStatefulset = "gamestatefulset"
+
+	// Pod CLB weight annotation key
+	AnnotationKeyForLoadbalanceWeight      = "tkex.bkbcs.tencent.com/clb-weight"
```

Comment:
Is it better to change to networkextension.bkbcs.tencent.com/clb-weight?

Reply:
yes, `networkextension.bkbcs.tencent.com/clb-weight` is better in this component. i'll fix it.